website/source/layouts/layout.erb

```diff
@@ -86,6 +86,7 @@
                   <li><a href="/api/index.html">API</a></li>
                   <li><a href="/community.html">Community</a></li>
                   <li><a href="https://www.hashicorp.com/products/nomad/?utm_source=oss&utm_medium=header-nav&utm_campaign=nomad">Enterprise</a></li>
+                  <li><a href="https://demo.nomadproject.io">Demo</a></li>
```

Comment:
UI Demo?

Reply:
"Demo" is consistent with the Consul site, but I agree that "UI Demo" is less vague. Will update.